### PR TITLE
docs: split up getting started guide into multiple pages

### LINKED ISSUE #2086: Docs: Consider splitting `Getting Started` page
Content could become more accessible with sub-sections.

Example:
 - Installation & Prerequisites
 - Usage (import and CDN)
 - Connecting to testnet
 - Connecting to a local node
 - Troubleshooting?

### PR DESCRIPTION
**Pull request by Dhaiwat10**
Closes #2086

### CODE REVIEW

**Comment by Dhaiwat10** [comment]
@arboleya I saw you mentioned a 'troubleshooting' page in the original issue. What would you want on this page? Do you have any rough idea/expectations?

**Comment by arboleya** [comment]
> [!IMPORTANT]
> Brainstorming.

@Dhaiwat10 At first, I thought about having a `fuels troubleshoot` command that could:
 1. check for version mismatches
 1. check ts settings ([example](https://github.com/FuelLabs/fuels-ts/issues/1907#issuecomment-2012591100))
 1. what else?

Like a utility, it could grow with time, incorporating handy validations and gotchas—think of a `brew doctor`. Maybe it's overkill, but maybe not. Anyway, in this rationale, the `troubleshoot` doc page could be a guide about using the new command (which requires implementing the command) or simply a guide to the manual procedures one could use to double-check one's setup.

What do you think?

File: apps/docs/.vitepress/config.ts [hunk 1]
```diff
@@ -45,7 +45,30 @@ export default defineConfig({
...
+              },
+              {
+                text: 'Connecting to a Local Node',
+                link: '/getting-started/connecting-to-a-local-node',
+              },
+              {
+                text: 'Further Resources',
+                link: '/getting-started/further-resources',
+              },
+            ],
```
[the remark this answers is not in the document]
Comment (arboleya) [reply apps/docs/.vitepress/config.ts:71]: Yeah, I agree.

[the remark this answers is not in the document]
Comment (arboleya) [reply apps/docs/.vitepress/config.ts:71]: Oh wait, but they are already, no?

Or are you talking about the `/guide/` in the path?

File: apps/docs/src/getting-started/installation-and-prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<script setup>
+  import { data } from '../versions.data'
+  const { fuels } = data
+</script>
+
+# Installation and Prerequisites
```
[the remark this answers is not in the document]
Comment (Dhaiwat10) [reply apps/docs/src/getting-started/installation-and-prerequisites.md]: https://github.com/FuelLabs/fuels-ts/pull/2202/commits/61c25ee90218afaae5cf4fbd3d9b2d97a11adaa2

File: apps/docs/.vitepress/config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Dhaiwat10) [reply apps/docs/.vitepress/config.ts]: https://github.com/FuelLabs/fuels-ts/pull/2202/commits/5ee781df51f82a2afca3414776edb446af73add8

Let's see if this is the fix

**Comment by Dhaiwat10** [comment]
@arboleya I have filed an issue for the `troubleshoot` command. I think we should just implement it and add the docs along with it when we do it, rather than doing the docs twice.

**Review by arboleya — APPROVED** [review]

**Review by Torres-ssf — APPROVED** [review]

**Review by danielbate — APPROVED** [review]

**Pull request merged**